packages/flutter_widgets/example/lib/notifications_service.dart

```diff
@@ -56,7 +56,7 @@ void initNotifications(Client client) {
     if (connector.token.value != null) {
       client.addDevice(
         connector.token.value,
-        Platform.isAndroid ? 'firebase' : 'apn',
+        Platform.isAndroid ? PushProvider.firebase : PushProvider.apn,
```

Comment:
I think converting this into a global getter will be better, something like `getPlatformPushProvider`

Reply:
just in the example or in the sdk?
because actually you might want to use firebase on ios too

Reply:
nvm then